### PR TITLE
LTE test harness

### PR DESCRIPTION
**Pull request by SrikanthPagadarai**

### CODE REVIEW

File: test/LTE_Tests/ADIRFSOM_LTETests.m [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+                testCase.TestSettings.RxGainMode;
+            testCase.Rx.kernelBuffersCount = 1;
+            testCase.Rx.SamplesPerFrame = ...
+                testCase.setRxSamplesPerFrame(testCase.BW);
+            
+            % configure custom filter settings
+            switch (testCase.BW)
+                case '5MHz'
+                    testCase.Tx.CustomFilterFileName = ...
+                        fullfile(testCase.root, '\ad936x_filters\LTE5_MHz.ftr');
```
Comment (tfcollins) [inline test/LTE_Tests/ADIRFSOM_LTETests.m:56]: These paths will all fail on Linux. Split at each \

File: test/LTE_Tests/Pluto_LTETests.m [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+                testCase.TestSettings.RxGainMode;
+            testCase.Rx.kernelBuffersCount = 1;
+            testCase.Rx.SamplesPerFrame = ...
+                testCase.setRxSamplesPerFrame(testCase.BW);
+            
+            % configure custom filter settings
+            switch (testCase.BW)
+                case '5MHz'
+                    testCase.Tx.CustomFilterFileName = ...
+                        fullfile(testCase.root, '\ad936x_filters\LTE5_MHz.ftr');
```
Comment (tfcollins) [inline test/LTE_Tests/Pluto_LTETests.m:56]: Same comment as above in regards to \

File: test/LTE_Tests/runLTETests.m [hunk 3]
```diff
@@ -0,0 +1,13 @@
+import matlab.unittest.TestSuite
+import matlab.unittest.TestRunner
+import matlab.unittest.plugins.XMLPlugin
+
+at = {'Pluto_LTETests', 'ADIRFSOM_LTETests'};
```
Comment (tfcollins) [inline test/LTE_Tests/runLTETests.m:5]: Did you want to add some filtering here?

**Review by tfcollins — COMMENTED** [review]
I think this needs to be rebased first with master

File: test/LTE_Tests/runLTETests.m (hunk 3 above)
Comment (SrikanthPagadarai) [reply test/LTE_Tests/runLTETests.m:5]: All issues have been addressed in pull request #62.

**Pull request closed without merging**